lib/btrfs_test.pm

```diff
@@ -64,7 +73,7 @@ it means that DBus got activated somehow, thus invalidated `snapper --no-dbus` t
 =cut
 sub snapper_nodbus_restore {
     if (script_run('systemctl is-active dbus')) {
-        script_run('systemctl default', 0);
+        script_run('systemctl default', 20);
         my $tty = get_root_console_tty;
         assert_screen "tty$tty-selected";
```

Comment:
if default timeout for assert_screen is not enough, we should increase it on this line rather than introducing timeout on `script_run` when changing systemd targets.

Reply:
But it is not the `assert_screen` which is not enough. it is the value in `script_run` which overrides the default_timeout. I do not know why this is set to 0 but it made my VRs fail.

Reply:
It is on purpose set to 0, so script_run does not wait for a return code as the SUT is changing a target. I prefer to keep it as it is and if necessary increase assert_screen timeout. The problem you talk about occurs only on PV.

Reply:
ready for review